tests/transform_dialect/cpu/fmaxf_16bit_expantion.mlir

```diff
@@ -0,0 +1,22 @@
+// RUN: iree-opt %s
```

Comment:
This shouldnt be in `tests/transform_dialect/cpu`. It should be in here
https://github.com/openxla/iree/tree/main/compiler/src/iree/compiler/Codegen/LLVMCPU/test with the other CPU passes test.

Please add this test to [this](https://github.com/openxla/iree/blob/8c41e5177f6a8cc33200bb91c8b01dee72bf0942/compiler/src/iree/compiler/Codegen/LLVMCPU/test/BUILD.bazel#L20) list and from your IREE source root directory you should run

```
build_tools/bazel_to_cmake/bazel_to_cmake.py
```

This will update the relevant CMake file.

Sorry for the hoops to jump through here...